### PR TITLE
Added 'debug' flag in values

### LINKED ISSUE #142: Add value to enable debug logging inside cluster
Into values.yaml add something that will set DEBUG=1 env variable for deployment container

### PR DESCRIPTION
**Pull request by OmAximani0**
- Helps to set environment on debug mode
- Can fix #142

### CODE REVIEW

File: charts/helm-dashboard/templates/deployment.yaml [hunk 1]
```diff
@@ -40,6 +40,12 @@ spec:
             value: /opt/dashboard/helm/config
           - name: HELM_DATA_HOME
             value: /opt/dashboard/helm/data
+          - name: DEBUG
+            {{- if .Values.debug }}
+            value: 1
+            {{- else }}
+            value: 0
```
Comment (undera) [inline charts/helm-dashboard/templates/deployment.yaml:47]: AFAIK any non-empty value would enable debugging. So intead of `0` it should be empty string. Also I'd enclosse `1` into quotes to make it YAML string

**Comment by undera** [comment]
Nice and clean, just address my small comment, please

**Comment by undera** [comment]
Amazing, I'll take it from here. 
Thanks for contributing!

**Comment by OmAximani0** [comment]
Thank you from

> Amazing, I'll take it from here. Thanks for contributing!

Thank you too for guiding me, I just came here by watching @kunal-kushwaha video https://www.youtube.com/watch?v=hYeNz3M4wT4 for a swag and did a contribution  😎  , Life is unpredictable 😜

**Pull request merged**